deskpro/deskpro-admin-frontend
# Uses defaultShow to define what columns initially show

Testing steps
- Load any table page
- See that some columns don't show
- See 'View' dropdown to see the columns initially hidden.
